Fast and Accurate GPU Quantization for Transformers.md

```diff
@@ -434,11 +430,7 @@ Whilst there might not be huge throughput improvements from FP16 output, there a
 
 - 50% less data loaded in the next kernel (now FP16 instead of INT32)
 - Avoid fusion of the dequantize operator with the next kernel
-- Avoid loading the dequantization vector in the next kernel (which CUTLASS pipelines the loading of)
```

Comment:
Is there a way we can avoid saying "next kernel" in each of these? Feels a bit repetitive